src/Server/Server.ts

```diff
@@ -157,6 +155,17 @@ export class Server implements IServer {
           log.error(`can't start server`, error)
           reject(error)
         })
+
+      if (process.env.NODE_ENV === 'development') {
```

Comment:
No need for this check.

Reply:
Done, but isn't it better if we keep it unexposed in production?

Reply:
I don't see a problem with this being in production, but if you prefer to disable it in prod, it should be done using a config.